src/components/activation-code/activation-code-activate.jsx

```diff
@@ -3,7 +3,7 @@ import { useTranslation } from "react-i18next";
 import { useNavigate } from "react-router-dom";
 import Form from "react-bootstrap/Form";
 import { Button } from "react-bootstrap";
-import { usePostV1UserActivationCodesActivateMutation } from "../../redux/api/api.generated";
+import { usePostV1UserActivationCodesActivateMutation } from "../../redux/api/api.generated.ts";
```

Comment:
Does this (adding `.ts`) really change anything?

Reply:
Otherwise it could not resolve the path.
The change is from using the .js file to using the .ts file (since I deleted the .js file).